packages/@react-spectrum/dnd/docs/dragAndDrop.mdx

```diff
@@ -0,0 +1,246 @@
+{/* Copyright 2022 Adobe. All rights reserved.
```

Comment:
I think we could just call this `dnd.mdx`, like we have `ssr.mdx`

Reply:
Sounds good, I'll do this last so the comments don't get outdated